src/templates/next/components/InfoCards/InfoCards.tsx

```diff
@@ -0,0 +1,104 @@
+import { InfoCardsProps } from "~/common"
+import Card from "../shared/Card"
+import { Heading } from "../../typography/Heading"
+
+const TitleSection = ({
+  title,
+  subtitle,
+  className = "",
+}: {
+  title: InfoCardsProps["title"]
+  subtitle: InfoCardsProps["subtitle"]
+  className?: string
+}) => {
+  return (
+    <div className={`flex flex-col gap-8 self-start max-w-3xl ${className}`}>
+      <h3 className={Heading[3]}>{title}</h3>
```

Comment:
need to add `text-content-strong` too! `Heading[3]` only provides the typography styles not colour 

![Screenshot 2024-03-21 at 10 26 07 PM](https://github.com/isomerpages/isomer-components/assets/77665182/4aa5dc66-befe-4a91-8712-7f1df131f43e)

Reply:
Fixed in https://github.com/isomerpages/isomer-components/pull/38/commits/7e78290ec0e3bb89f63595adf4b182f32ca8bbf7!